pipelines/controller/src/main/java/org/openmrs/analytics/PipelineManager.java

```diff
@@ -304,6 +312,64 @@ synchronized void runIncrementalPipeline()
     }
   }
 
+  /**
+   * This method periodically checks on Thrift server output directory and commands to create
+   * resource tables. The very first run happens over a minute later once controller spins up. And
+   * it's programmed to check on Thrift server every hour in case it went down for unfortunate
+   * reasons.
+   *
+   * @throws IOException
+   * @throws SQLException
+   */
+  @Scheduled(initialDelay = 60000, fixedDelay = 3600000)
```

Comment:
Can we also make this configurable via application.yml file to be more flexible?

Reply:
Let's remove the scheduled option for now (I know I suggested that myself). Given that we are fixing the Metastore persistence issue in #654 I don't think this is needed anymore and it can cause other issues. So we only check and create Metastore tables on restart and after each pipeline run (but not periodically).

Reply:
ack